nixos/modules/services/networking/wireguard.nix

```diff
@@ -391,6 +414,7 @@ in
           message = "networking.wireguard.interfaces.${interfaceName} peer «${peer.publicKey}» has both presharedKey and presharedKeyFile set, but only one can be used.";
         }) all_peers;
 
+    networking.firewall.checkReversePath = "loose";
```

Comment:
Would every wireguard user want this or is this only relevant to people trying to do a 0.0.0.0/0 setup?

Reply:
A good question.

Reply:
Reverse path filtering ensures some routing symmetry, meaning you can only receive packets from a src addr on an interface, when you would also use that interface to send to that address.

- This is not strictly necessary for every wireguard setup.
- It is probably necessary in this case, because you have two default routes in place.

Reply:
loose reverse path filters are the default for most systemd-based distros. I guess we can drop that line, and instead let https://github.com/NixOS/nixpkgs/pull/66482 handle this.